test/validate.spec.js

```diff
@@ -0,0 +1,28 @@
+'use strict';
+
+const { run, assert } = require('./util');
+const { releaseNotes1 } = require('./assets');
+
+describe('validate', () => {
+  it('should succeed when no file is specified and ./release-notes.yml exists', () => {
+    return assert.isFulfilled(run('validate'));
+  });
+  it('should succeed when valid file is specified', () => {
+    return assert.isFulfilled(run(`validate ${releaseNotes1.yamlPath}`));
+  });
+  /* it('should fail when no file is specified but file is invalid', () => {
```

Comment:
I'd prefer skipping tests, that test an unimplemented feature.

Reply:
Are you saying that specifying an explicit file is not supported yet? And this test case is in fact invalid and produced a false positive?

Reply:
No, sorry i don't know where i was with my thoughts.

Passing an explicite file is supported. And this test or its setup is simply not fully implemented.
I guess you wanted to test if the command fails properly when the default `release-notes.yml` file in the current working directory is not valid.
I think we should either drop this commented out test case or provide the required setup for it.

Reply:
Alright, I've removed the out-commented test